extensions/amp-story/1.0/amp-story-page-attachment.js

```diff
@@ -133,25 +195,61 @@ export class AmpStoryPageAttachment extends DraggableDrawer {
    * @override
    */
   open(shouldAnimate = true) {
+    if (this.state_ === DrawerState.OPEN) {
+      return;
+    }
+
     super.open(shouldAnimate);
 
     this.storeService_.dispatch(Action.TOGGLE_SYSTEM_UI_IS_VISIBLE, false);
 
-    const currentHistoryState = /** @type {!Object} */ (getState(
-      this.win.history
-    ));
-    const historyState = {
-      ...currentHistoryState,
-      [HistoryState.ATTACHMENT_PAGE_ID]: this.storeService_.get(
-        StateProperty.CURRENT_PAGE_ID
-      ),
-    };
-
-    this.historyService_.push(() => this.closeInternal_(), historyState);
+    if (this.type_ === AttachmentType.INLINE) {
+      const currentHistoryState = /** @type {!Object} */ (getState(
```

Comment:
Seems asymmetrical to have `openRemote_` but then not `openInline_`; WDYT about extracting this to a separate function?

Reply:
I don't feel strongly here, but `openInline_` would be confusing as the actual opening logic happens in the `super.open()`. It would just push a new history entry. Maybe the `if` statement itself is confusing, it'd be better if it was something like `shouldCreateANewHistoryEntry`.
I changed the `if` to `NOT remote` and added a comment to maybe make this more clear, but let me know if you want this in a separate method.